getnacelle/nuxt-reference-store
# About + Contact Page

### WHY are these changes introduced?

Ticket [[LD-1104](https://nacelle.atlassian.net/browse/LD-1105)] 

### WHAT is this pull request doing?

- Adds about + contact page components: `HeroBanner`, `SideBySide`, `PromoStrip`, `CtaStrip`, `TeamBios`
- Adds dynamic content from Sanity

### How to Test

1. Checkout branch `LD-1111_content`
2. `npm run dev` to start Nuxt
3. Open the site to `/pages/about` and `/pages/contact` to see sections filled with content from Sanity.
